upstream-community-operators/kubeflow/1.2.0/kubeflow.v1.2.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,287 @@
+                    "repoRef": {
+                        "name": "manifests",
+                        "path": "metacontroller/base"
+                    }
+                  },
+                  "name": "metacontroller"
+              },
+              {
+                  "kustomizeConfig": {
+                    "repoRef": {
+                        "name": "manifests",
+                        "path": "stacks/ibm/application/metadata"
+                    }
+                  },
+                  "name": "metadata"
+              },
+              {
+                  "kustomizeConfig": {
+                    "repoRef": {
+                        "name": "manifests",
+                        "path": "stacks/ibm/application/spark-operator"
+                    }
+                  },
+                  "name": "spark-operator"
+              },
+              {
+                  "kustomizeConfig": {
+                    "repoRef": {
+                        "name": "manifests",
+                        "path": "knative/installs/generic"
+                    }
+                  },
+                  "name": "knative"
+              },
+              {
+                  "kustomizeConfig": {
+                    "repoRef": {
+                        "name": "manifests",
+                        "path": "kfserving/installs/generic"
+                    }
+                  },
+                  "name": "kfserving"
+              },
+              {
+                  "kustomizeConfig": {
+                    "repoRef": {
+                        "name": "manifests",
+                        "path": "stacks/ibm/application/spartakus"
+                    }
+                  },
+                  "name": "spartakus"
+              }
+            ],
+            "repos": [
+              {
+                  "name": "manifests",
+                  "uri": "https://github.com/kubeflow/manifests/archive/v1.2.0.tar.gz"
+              }
+            ],
+            "version": "v1.2.0"
+        }
+      }
+      ]
+    capabilities: Basic Install
+    categories: "AI/Machine Learning"
+    description: "Kubeflow Operator for deployment and management of Kubeflow"
+    support: Kubeflow
+    repository: https://github.com/kubeflow/kfctl
+    createdAt: '2020-03-19T00:00:00Z'
+    containerImage: aipipeline/kubeflow-operator:v1.2.0
+    certified: 'False'
+  name: kubeflow.v1.2.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: KfDef is the Schema for the applications API
+      kind: KfDef
+      name: kfdefs.kfdef.apps.kubeflow.org
+      version: v1
+      displayName: Kubeflow
+      group: kfdef.apps.kubeflow.org
+  description: "Kubeflow Operator for deployment and management of Kubeflow components. Applicable for Kubeflow versions 1.0 and above. Check [Kubeflow Operator documentation](https://github.com/kubeflow/kfctl/blob/master/operator.md) for more details."
```

Comment:
Did you remove the [`description`](https://github.com/operator-framework/community-operators/blob/master/upstream-community-operators/kubeflow/1.0.0/kubeflow.v1.0.0.clusterserviceversion.yaml#L27) part from v1.0.0's csv? Was that the intention? If not, could you make sure that is still there? And then there are some changes required after that.